keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,321 @@
+https://github.com/kubernetes/sig-release/issues/1494.
+
+## Proposal
+
+### User Stories
+
+Kubernetes releases are made by real people. The technical aspects—for example,
+the release automation—reflects only a tiny part of the complete cycle. This
+means we will mainly focus on the human aspects and their corresponding roles
+when deciding to move to a 3-releases-per-year cadence.
+
+#### End User
+
+Most companies are facing issues upgrading Kubernetes 4 times a year. Providing
+only 3 releases per year will relax this situation.
+
+#### Distributors and downstream projects
+
+Downstream projects assemble their solution from different projects. Having
+fewer upgrades helps them to reduce complexity. For example, cloud providers
+will gain more room for upgrading their infrastructure.
+
+#### Contributors
+
+With a lower release cadence, contributors will gain more time for project
+enhancements, feature development, planning, and testing. It will provide more
+room for maintaining their mental health and prepare for events like KubeCon.
+
+Through this proposal SIG Release's aim is to give contributors more
+flexibility to decide how to invest their time. It is explicitly *not* to push
+contributors in doing more.
+
+#### SIG Release members
+
+By applying a cadence of 3 releases per year, SIG Release members will gain a
+reduced management overhead. There are also only 3 patch releases to maintain,
+which right now can overlap up to 4. SIG Release will gain more time to ensure
+a seamless transition from the previous release team to the next one. It is
+also possible to include more shadows if the role leads conclude that this is
+appropriate.
+
+### Risks and Mitigations
+
+#### Concentrating risk
+
+In theory a reduced release cadence will cause more changes for every release.
+This means that there will be an increased risk, which would usually be split
+up into 4 dedicated milestones rather than 3.
+
+SIG Release cannot mitigate this risk directly, but is able to track and
+influence it during each release cycle. It's the responsibility of SIG Release,
+together with SIG Testing and SIG Architecture, to identify new gaps and issues
+in the release cadence and mitigate them on a case-by-case basis.
+
+#### Attention to tests
+
+This KEP does not propose any change to the release cycle itself and assumes
+that the same periods for Code and Test Freeze. Assuming that, there is an
+increased risk for flakes and test failures. It will be the responsibility of
+SIG Release to mitigate this, together with the CI signal role. If we speak
+about an overall release cycle enhancement of 3-4 weeks, then we believe that
+SIG Release is able to mitigate this risk over multiple releases.
+
+#### Attention to dependencies
+
+Having fewer releases will introduce the risk of missing dependencies—for
+example, Golang upgrades. This has to be mitigated on a case-by-case basis, in
+the same way as it is being done right now.
```

Comment:
yeah, this seems orthogonal to the release cadence, just random

Reply:
Alright, I think in any case we cannot be more specific here. Let me resolve this discussion and feel free to come back if you think this needs more refinement.